docs/source/actions.rst

```diff
@@ -410,6 +410,57 @@ The ``merge`` action merges the pull request into its base branch. The
          itself as the commit message. The pull request number will be added to
          end of the title.
 
+queue
+=====
+
+|premium plan tag|
+
+The ``queue`` action moves the pull request into one of the merge queue defined
+in ``queue_rules``.. The ``queue`` action takes the following parameter:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1 3
+
+   * - Key Name
+     - Value Type
+     - Default
+     - Value Description
+   * - ``name``
+     - string
+     -
+     - The name of the merge queue to move pull request to.
+   * - ``strict_method``
+     - string
+     - ``merge``
+     - Update method to use to update the pull request with its base branch
+       when :ref:`strict merge` is enabled. Possible values:
+
+       * ``merge`` to merge the base branch into the pull request.
+       * ``rebase`` to rebase the pull request against its base branch.
+
+       Note that the ``rebase`` method has many drawbacks, see :ref:`strict
+       rebase`.
+   * - ``priority``
+     - 1 <= integer <= 10000 or ``low`` or ``medium`` or ``high``
+     - ``medium``
+     - This set the priority of the pull request in the queue when ``smart``
+       :ref:`strict merge` is enabled. The pull request with the highest priority is merged first.
+       ``low``, ``medium``, ``high`` are aliases for ``1000``, ``2000``, ``3000``.
+   * - ``commit_message``
+     - string
+     - ``default``
+     - Defines what commit message to use when merging using the ``squash`` or
+       ``merge`` method. Possible values are:
+
+       * ``default`` to use the default commit message provided by GitHub
+         or defined in the pull request body (see :ref:`commit message`).
+
+       * ``title+body`` means to use the title and body from the pull request
+         itself as the commit message. The pull request number will be added to
+         end of the title.
+
```

Comment:
I'm a bit confused as to why those merge options are now per queue and per PR.
Is there any reason do move them here?

Reply:
I just forget to remove `strict-method` from the doc here.

Reply:
The list has been split